dvc/main.py

```diff
@@ -52,7 +52,7 @@ def main(argv=None):  # noqa: C901
 
         with debugtools(args):
             cmd = args.func(args)
-            ret = cmd.run()
+            ret = cmd.do_run()
```

Comment:
I do like `run` for aesthetics. Maybe I should do metaclass hacks here to keep `run`. I'm feeling lazy to change other subclasses.